eos-tech-support/eos-check-secureboot

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+# Check if we are under secure boot by reading the correspondent EFI var
+
+EFIVARS = "/sys/firmware/efi/vars"
+SECUREBOOT ="SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c"
+
+f = open(EFIVARS + "/" + SECUREBOOT + "/data", "rb");
+secure_boot = ord(f.read(1))
```

Comment:
I did a bit of research yesterday and did not find a way to read a binary file in bash that looked reliable. I can spend a bit more of time on it if you want.